tests/openassetio-core-c/private/InfoDictionaryTest.cpp

```diff
@@ -303,17 +305,17 @@ struct SuiteAccessorFixture<openassetio::Str> : InfoDictionaryFixture {
   OPENASSETIO_NS(StringView) actualValue_ = kInitialValue;
 };
 
-TEMPLATE_TEST_CASE_METHOD(SuiteAccessorFixture, "InfoDictionary accessed via C API", "",
+TEMPLATE_TEST_CASE_METHOD(AccessorFixture, "InfoDictionary accessed via C API", "",
                           openassetio::Bool, openassetio::Int, openassetio::Float,
                           openassetio::Str) {
-  GIVEN("a populated C++ InfoDictionary and its C handle and suite function pointer") {
-    using Fixture = SuiteAccessorFixture<TestType>;
+  GIVEN("a populated C++ InfoDictionary and its C handle function pointer") {
+    using Fixture = AccessorFixture<TestType>;
     // Map constructed with some initial data.
     auto& infoDictionary = Fixture::infoDictionary_;
     // Opaque handle to map.
     const auto& infoDictionaryHandle = Fixture::infoDictionaryHandle_;
     // Suite function for type under test.
```

Comment:
[consider] Here, and other places, Is it still a suite function, or just function?

Reply:
Updated erroneous comments in a486fe97 - most swapped "suite" with "C API" (which handily has the same number of characters).